primo-explore/custom/main.js

```diff
@@ -639,15 +633,23 @@ app.component('prmFullViewAfter', {
 
 // End Altmetric plugin
 
-// Add the chat button
-//
-// (function () {
-//   var lc = document.createElement('script');lc.type = 'text/javascript';lc.async = 'true';
-//   lc.src = ('https:' == document.location.protocol ? 'https://' : 'http://') + 'northwestern.libanswers.com/load_chat.php?hash=8479f851eb63bff929998c6128b41b20';
-//   var s = document.getElementsByTagName('script')[0];s.parentNode.insertBefore(lc, s);
-//
-//   // End the chat button
-// })();
+// LibChat button
+app.component('libChatComponent', {
+  template: `<div id="libchat_button">
+               <div id="libchat_4e6bc50e0cea1aa526c386eb796636d7"></div>
+             </div>`
+});
+
+app.component('prmExploreFooterAfter', {
+  bindings: { parentCtrl: `<` },
+  template: `<lib-chat-component></lib-chat-component>`
+});
+
+(function () {
+  var lc = document.createElement('script');lc.type = 'text/javascript';lc.async = 'true';
+  lc.src = ('https:' == document.location.protocol ? 'https://' : 'http://') + 'galter-northwestern.libanswers.com/load_chat.php?hash=4e6bc50e0cea1aa526c386eb796636d7';
```

Comment:
What is "4e6bc50e0cea1aa526c386eb796636d7" ? I'm guessing it's an API key/token or something similar. Personally I think this kind of thing is nice to throw in a variable with a descriptive name, but maybe it's clear what's going on if you look into the LibChat documentation.

Reply:
agreed